plasma/root_chain/contracts/RootChain/RootChain.sol

```diff
@@ -115,7 +115,15 @@ contract RootChain {
         bytes32 root = childChain[blknum].root;
         bytes32 depositHash = keccak256(msg.sender, amount);
         require(root == depositHash);
-        addExitToQueue(depositPos, msg.sender, amount);
+        addExitToQueue(depositPos, msg.sender, amount, childChain[blknum].created_at);
+    }
+
+    function startFeeExit(uint256 amount)
+        public
+        isAuthority
+        returns (uint256)
+    {
+        addExitToQueue(0, msg.sender, amount, block.timestamp + 1);
```

Comment:
Are we ok with only allowing a single fee exit at a time (because the `utxoPos` is always 0)? Also, this leads to a problem in `finalizeExits` because we `delete exits[utxoPos].owner;` and not the full exit, so the operator can only ever exit once. Can solve naively by incrementing the fee exit by 1 each time.